Ball of Duty Server/Program.cs

```diff
@@ -13,25 +14,63 @@ namespace Ball_of_Duty_Server
 {
     internal class Program
     {
+        public enum Command
+        {
+            QUIT,
+            WRITE_ACTIVE_PLAYERS,
+            BAN,
+            SERVER_MESSAGE
+        }
+
+
+        public static Command? GetCommandType(String line)
+        {
+            if (line.ToLower().StartsWith("sm") || line.ToLower().StartsWith("server_message"))
+            {
+                return Command.SERVER_MESSAGE;
+            }
+            else if (line.ToLower().StartsWith("quit"))
+            {
+                return Command.QUIT;
+            }
+            else if (line.ToLower().StartsWith("active_players"))
+            {
+                return Command.WRITE_ACTIVE_PLAYERS;
+            }
+            else if (line.ToLower().StartsWith("ban"))
+            {
+                return Command.BAN;
+            }
+            else
+            {
+                return null;
```

Comment:
you should always define a default value in an enum so you avoid using nullables or other silly stuff for enum values.
Command.NONE = 0